hack/make-rules/test-cmd-util.sh

```diff
@@ -3707,7 +3707,7 @@ run_RESTMapper_evaluation_tests() {
   # Pre-condition: None
   # Command
   kubectl get "${kube_flags[@]}" unknownresourcetype 2>${RESTMAPPER_ERROR_FILE} || true
-  if grep -q "the server doesn't have a resource type" "${RESTMAPPER_ERROR_FILE}"; then
+  if grep -q "no matches for kind \"unknownresourcetype\"" "${RESTMAPPER_ERROR_FILE}"; then
```

Comment:
This message is technically incorrect - we couldn't find a resource or a kind called unknown resource type, and we are *very* careful in the CLI to describe this argument as a resource argument.  I think you need to fix the message in the case where you fall all the way through to be consistent with the previous release.

Reply:
Thanks, added an error message when this happens to maintain consistency with previous behavior